app/views/main/index.html

```diff
@@ -6,6 +6,8 @@
     <meta name="theme-color" content="#242d33">
 
     <link rel="stylesheet" href="style/main.css">
+    <script async src="../../shared/js/promises.js"></script>
+    <script async src="../shared/js/utils.js"></script>
```

Comment:
I hesitated to do that, because it will slowdown the app startup a bit I guess. WDYT?